hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/blockmanagement/DatanodeDescriptor.java

```diff
@@ -197,6 +197,9 @@ public Type getType() {
   /** A queue of blocks to be replicated by this datanode */
   private final BlockQueue<BlockTargetPair> replicateBlocks =
       new BlockQueue<>();
+  /** A queue of ec blocks to be replicated by this datanode. */
+  private final BlockQueue<BlockTargetPair> ecReplicatedBlocks =
+          new BlockQueue<>();
   /** A queue of blocks to be erasure coded by this datanode */
   private final BlockQueue<BlockECReconstructionInfo> erasurecodeBlocks =
       new BlockQueue<>();
```

Comment:
I prefer more descriptive variable names. What do you think?
```suggestion
  /** A queue of ec blocks to be replicated by this datanode. */
  private final BlockQueue<BlockTargetPair> ecBlocksToBeReplicated =
          new BlockQueue<>();
  /** A queue of ec blocks to be erasure coded by this datanode */
  private final BlockQueue<BlockECReconstructionInfo> ecBlocksToBeErasureCoded =
      new BlockQueue<>();
```

Reply:
Yes, this way the naming is clearer.